x-pack/plugins/canvas/public/components/workpad_header/share_menu/share_menu.tsx

```diff
@@ -45,12 +46,14 @@ export const ShareMenu = () => {
   const ReportingComponent =
     ReportingPanelPDFComponent !== null
       ? ({ onClose }: { onClose: () => void }) => (
-          <ReportingPanelPDFComponent
-            getJobParams={() => getPdfJobParams(sharingData, platformService.getKibanaVersion())}
-            layoutOption="canvas"
-            onClose={onClose}
-            objectId={workpad.id}
-          />
+          <KibanaErrorBoundary>
+            <ReportingPanelPDFComponent
+              getJobParams={() => getPdfJobParams(sharingData, platformService.getKibanaVersion())}
+              layoutOption="canvas"
+              onClose={onClose}
+              objectId={workpad.id}
+            />
+          </KibanaErrorBoundary>
```

Comment:
Tested this by:
1. blocking requests that match `` from loading in the browser
2. Go to Canvas and open a workpad
3. Attempt to create a PDF report of the workpad